qutebrowser/config/config.py

```diff
@@ -330,6 +330,8 @@ class ConfigManager(QObject):
     CHANGED_OPTIONS = {
         ('content', 'cookies-accept'):
             _get_value_transformer('default', 'no-3rdparty'),
+        ('storage', 'download-directory'):
+            _get_value_transformer('', '%'),
```

Comment:
I think this will change `''` to `'%'` unconditionally, i.e. there'll be no way to set `''` anymore.

This should really only be used for values which make no sense anymore.